userspace/libsinsp/filterchecks.h

```diff
@@ -231,6 +231,44 @@ friend class chk_compare_helper;
 // Filter check classes
 ///////////////////////////////////////////////////////////////////////////////
 
+//
+// fd checks
+//
+class sinsp_filter_check_fspath : public sinsp_filter_check
+{
+public:
+	enum check_type
+	{
+		TYPE_NAME = 0,
```

Comment:
Preference so we can change order more easily in the future if we want.

```suggestion
		TYPE_NAME = 0,
		TYPE_NAMERAW,
		TYPE_SOURCE,
		TYPE_SOURCERAW,
		TYPE_TARGET,
		TYPE_TARGETRAW,
```

Reply:
Looking at other enums in filterchecks.h, around 1/3 define explicit values and 2/3 do not. I agree with jason that explicit values are very slightly better.

But as jason alluded to, the real problem is that the field enums are defined in the header file and the descriptions are in a separate array that has to have the same exact order. I would like to change that someday, but we should change all of them at the same time when we do it.